link/src/main/java/com/stripe/android/link/ui/signup/SignUpScreen.kt

```diff
@@ -126,22 +128,27 @@ internal fun SignUpBody(
             style = MaterialTheme.typography.body1,
             color = MaterialTheme.colors.onSecondary
         )
-        EmailCollectionSection(
-            enabled = true,
-            emailElement = emailElement,
-            signUpState = signUpState
-        )
+        PaymentsThemeStatic {
```

Comment:
Can we call this at the top of this method and apply to all PaymentsTheme views?

Reply:
Unfortunately not here. We need to theme the top level elements differently. Here's what it looks like with `DefaultPaymentsTheme` applied: 
![lostLinkStyling](https://user-images.githubusercontent.com/89166418/169158789-073c5969-0f55-4331-84bb-60866de124b2.png)

Reply:
Hmm, okay, I think because PaymentsTheme overrides some MaterialTheme values, and we use those.